bin/get_snapshot.cmd

```diff
@@ -0,0 +1,101 @@
+@ECHO OFF
+setlocal enabledelayedexpansion
+
+set branch=%1
+set build=%2
+set cpu=%3
+set revision=%4
+
+REM Check if parameters are set
+if %branch%.==. GOTO args_error
+if %build%.==. GOTO args_error
+if %cpu%.==. GOTO args_error
+if %revision%.==. (
+	GOTO args_error
+) else (
+	GOTO set_env
+)
+
+:args_error
+echo User error: must specify branch, build type, CPU arch and revision code
+echo get_release "<branch> <build> <cpu> <revision code | latest>"
+echo Branch can be any of: 5.6, 7.0, 7.1, 7.2, 7.3, 7.4
+echo Build can be any of: NTS, TS
+echo CPU can be any of: X64, X86
+echo Revision code starts with r (i.e. rxxxxxx)
+exit /b
+
+:set_env
+REM set important env vars
+IF DEFINED PFTT_SHELL GOTO :skip_set_env
+CALL %~dp0set_env.cmd
+:skip_set_env
+
+SET PHP_BUILDS=%~d0\PHPBuilds
+
+REM Create cache folder if it does not exist
+if not exist %PHP_BUILDS% (
+	md %~d0\PHPBuilds
+)
+
+REM Set file_name and test_pack based on parameters
+if %branch%==5.6 (
+	set file_name=php-!branch!-%build%-windows-vc11-%cpu%
+	set test_pack=php-test-pack-!branch!-%build%-windows-vc11-%cpu%
+) else if %branch%==7.0 (
+	set file_name=php-!branch!-%build%-windows-vc14-%cpu%
+	set test_pack=php-test-pack-!branch!-%build%-windows-vc14-%cpu%
+) else if %branch%==7.1 (
+	set file_name=php-!branch!-%build%-windows-vc14-%cpu%
+	set test_pack=php-test-pack-!branch!-%build%-windows-vc14-%cpu%
+) else if %branch%==7.2 (
+	set file_name=php-!branch!-%build%-windows-vc15-%cpu%
+	set test_pack=php-test-pack-!branch!-%build%-windows-vc15-%cpu%
+) else if %branch%==7.3 (
+	set file_name=php-!branch!-%build%-windows-vc15-%cpu%
+	set test_pack=php-test-pack-!branch!-%build%-windows-vc15-%cpu%
+) else if %branch%==7.4 (
+	set file_name=php-!branch!-%build%-windows-vs16-%cpu%
+	set test_pack=php-test-pack-!branch!-%build%-windows-vs16-%cpu%
+)
+
+if /i %revision%==latest goto get_latest_revision
```

Comment:
This could actually call `:get_latest_revision` [as subroutine](https://ss64.com/nt/call.html).

Reply:
Updated and moved `get_latest_revision` to another file and now calls it instead. It is much cleaner.